app/controllers/organisations_controller.rb

```diff
@@ -117,7 +117,6 @@ def reactivate
   end
 
   def update
-    selected_rent_periods = rent_period_params[:rent_periods].compact_blank
```

Comment:
Moved this to the bottom and under an if statement, because we're not using it until then and if there's any other update than rent periods, it fails (for example deactivating the org)